anvio/data/misc/PEOPLE/CONTRIBUTORS.yaml

```diff
@@ -209,3 +209,6 @@
       inst: Department of Microbiology - The Ohio State University
       inst_link: https://microbiology.osu.edu/
       current: true
+
+- github: treitlis
+  name: Sebastian Treitli
```

Comment:
name: Sebastian Cristian Treitli
  email: sebastian.treitli@gmail.com
  avatar: KatKananen.jpg
  orcid: 0000-0001-8000-1780
  bio: "Microbiologist and bioinformatician - interested in protists and symbiotic interactions between protists and bacteria."
  affiliations:
    - title: Post-Doctoral researcher
      inst: Max-Planck Institute for Terrestrial Microbiology
      inst_link: mpi-marburg.mpg.de/
      current: true